hydra_base/lib/attributes.py

```diff
@@ -119,6 +119,14 @@ def get_attribute_by_id(attr_id, **kwargs):
 
     return attr_i
 
+def get_attributes_by_id(attr_ids, **kwargs):
+    """
+        Get a list of specific attributes by their IDs.
+    """
+    attrs_i = db.DBSession.query(Attr).filter(Attr.id.in_(attr_ids)).all()
+
+    return attrs_i
```

Comment:
Can avoid the query on empty or `None` argument and just return the results otherwise...

```suggestion
    if not attr_ids:
        return []
            
    return db.DBSession.query(Attr).filter(Attr.id.in_(attr_ids)).all()
```